source/iostreams.tex

```diff
@@ -5916,6 +5916,19 @@
 \postcondition
 \tcode{rdbuf() == sb}.
 
+\indexlibrary{\idxcode{basic_ostream}!constructor}%
```

Comment:
Ah, and the other half of the mystery resolved! LWG 911 made these members protected, which is why they show up at the /end/ of the class definition - but the declarations are present and correct.